app/views/_volume-claim-templates.html

```diff
@@ -0,0 +1,46 @@
+<div class="pod-template-block">
+  <div
+    ng-repeat="template in templates"
+    class="pod-template">
+    <div class="component-label">Storage claim: {{template.metadata.name}}</div>
+
+    <div row class="pod-template-image icon-row">
+      <div>
+        <span class="fa fa-lock" aria-hidden="true"></span>
+      </div>
+      <div flex class="word-break">
+        <span class="pod-template-key">Access Modes:</span>
+        <span ng-repeat="mode in template.spec.accessModes">
+          {{mode | sentenceCase }}<span ng-if="!$last">, </span>
+        </span>
+      </div>
+    </div>
+
+    <div row class="pod-template-image icon-row">
+      <div>
+        <span class="fa fa-database" aria-hidden="true"></span>
+      </div>
+      <div flex class="word-break">
+        <span class="pod-template-key">Capacity:</span>
+        <span>
+          {{template.spec.resources.requests.storage}}
+        </span>
+      </div>
+    </div>
+
+    <div
+      row class="pod-template-image icon-row"
+      ng-if="template.spec.selector.matchLabels">
```

Comment:
ok looks like this is using the new type of selectors, we can do a follow-up PR tomorrow, but we should handle matchExpressions as well, can you make a TODO for that for tomorrow in your notebook :)